.github/workflows/release.yml

```diff
@@ -9,15 +9,21 @@ on:
       - master
       - "[2-9][0-9].[0-9][0-9].x"
     paths:
+      - "centreon-awie/**"
+      - "centreon-dsm/**"
+      - "centreon-gorgone/**"
+      - "centreon-ha/**"
       - "centreon-open-tickets/**"
+      - "centreon/**"
+  pull_request:
+    paths:
+      - ".github/workflows/release.yml"
```

Comment:
I think you can remove this trigger because release job will never run regarding the `if` condition